openhabian-setup.sh

```diff
@@ -59,7 +59,7 @@ OLDWD=$(pwd) && cd /opt
 if [[ -n "$UNATTENDED" ]]; then
   # apt/dpkg commands will not try interactive dialogs
   export DEBIAN_FRONTEND=noninteractive
-  apt-get update
+  apt_update
```

Comment:
Do we need to wait for it to finish here?

Reply:
not here but before calling apt-get next time which is in timezone routine